Libplanet/Action/ActionEvaluator.cs

```diff
@@ -605,7 +605,7 @@ internal static IImmutableSet<Address> GetUpdatedAddresses<T>(Transaction<T> tx)
                     tx.Id,
                     tx.Signer,
                     block.Index,
-                    ByteUtil.Hex(block.PreEvaluationHash));
+                    ByteUtil.Hex(block.PreEvaluationHash.ByteArray));
```

Comment:
Maybe just use `block.PreEvaluationHash` is more clean? (It implements `.ToString()` methods)

Reply:
That is `ByteUtil.Hex`, not `ByteUtil.ToHex`. Parameter type is `byte[]`.